assignments/this.js

```diff
@@ -13,14 +13,38 @@
 
 // code example for Window Binding
 
+console.log(this);
+
 // Principle 2
 
 // code example for Implicit Binding
 
+const friend = {
+    name: Jordan,
+    age: 30,
+    location: Vancouver
+    intro: function() {
+        return `${this.name} lives in ${this.location}, he's ${this.age} and he's my BEST friend!`;
+    }
+}
+
 // Principle 3
 
 // code example for New Binding
+function friends = {
+    name: this.name,
+    intro: function () {
+        return `${this.name} is my BEST friend!`;
+    }
+}
+
+const jesse = new friends('Jesse');
+const mikayla = new friends('Mikayla');
 
 // Principle 4
 
-// code example for Explicit Binding
\ No newline at end of file
+// code example for Explicit Binding
+
+const change = mikayla.intro.bind(jesse);
+
+console.log(change());
```

Comment:
Good work and well done with the application of the material.  Good examples for the examples of this as well.